doc/modules/motion_correction.rst

```diff
@@ -134,11 +134,11 @@ Low-level API
 -------------
 
 All steps (**activity profile**, **motion inference**, **motion interpolation**) can be launched with distinct functions.
-This can be useful to find the good method and finely tune/optimize parameters at every steps.
+This can be useful to find the best method and finely tune/optimize parameters at each step.
 All functions are implemented in the :py:mod:`~spikeinterface.sortingcomponents` module.
 They all have a simple API with SpikeInterface objects or numpy arrays as inputs.
 Since motion correction is a hot topic, these functions have many possible methods and also many possible parameters.
-Finding the good combination of method/parameters is not that easy, but it should be doable, assuming the presets are not
+Finding the best combination of method/parameters is not that easy, but it should be doable, assuming the presets are not
```

Comment:
Same here, "a good" vs "the best", don't know if the intent was the academic humble style.

Reply:
Same here, no need to be humble :P

Reply:
yes we need to be humble!

Reply:
In this case, you would want "best" because you're optimizing something. If you say "the good" it implies moral goodness like this is the religious combination. You could also say "right" combination instead if best feels too strong. But yeah best would be the standard for this type of situation.